### ISSUE TITLE
tornado.general:Write error on <httpretty.core.fakesock.socket object at 0x10c6459d0>: [Errno 57] Socket is not connected

### ISSUE THREAD

**Issue by dE1l** [gabrielfalcao/HTTPretty#402]
```
from tornado.httpclient import HTTPClient
import httpretty

httpretty.enable()
http_client = HTTPClient()
response = http_client.fetch("http://www.google.com/")
print(response.body)
```

> WARNING:tornado.general:Write error on <httpretty.core.fakesock.socket object at 0x10c6459d0>: [Errno 57] Socket is not connected
> Traceback (most recent call last):
>   File "/Users/rbeltsov/git/card_binder/tests/del_me.py", line 9, in <module>
>     response = http_client.fetch("http://www.google.com/")
>   File "/Users/rbeltsov/git/card_binder/venv/lib/python3.7/site-packages/tornado/httpclient.py", line 133, in fetch
>     functools.partial(self._async_client.fetch, request, **kwargs)
>   File "/Users/rbeltsov/git/card_binder/venv/lib/python3.7/site-packages/tornado/ioloop.py", line 532, in run_sync
>     return future_cell[0].result()
>   File "/Users/rbeltsov/git/card_binder/venv/lib/python3.7/site-packages/tornado/iostream.py", line 988, in _handle_write
>     num_bytes = self.write_to_fd(self._write_buffer.peek(size))
>   File "/Users/rbeltsov/git/card_binder/venv/lib/python3.7/site-packages/tornado/iostream.py", line 1169, in write_to_fd
>     return self.socket.send(data)  # type: ignore
>   File "/Users/rbeltsov/git/card_binder/venv/lib/python3.7/site-packages/httpretty/core.py", line 675, in send
>     return self.forward_and_trace('send', *args, **kwargs)
>   File "/Users/rbeltsov/git/card_binder/venv/lib/python3.7/site-packages/httpretty/core.py", line 666, in forward_and_trace
>     return callback(*a, **kw)
> OSError: [Errno 57] Socket is not connected

**Comment by gabrielfalcao** [comment]
thanks for reporting @dE1l, I'll set time to work on this soon.

**Comment by gabrielfalcao** [comment]
the fix will be available in the next release (1.0.4)
